SolrNet/Impl/SolrBasicServer.cs

```diff
@@ -140,6 +140,11 @@ public ExtractResponse SendAndParseExtract(ISolrCommand cmd)
             return extractResponseParser.Parse(xml);
         }
 
+        public ResponseHeader AtomicUpdate(string uniqueKey, string id, IEnumerable<AtomicUpdateSpec> updateSpecs, AtomicUpdateParameters parameters)
+        {
+            var atomicUpdate = new AtomicUpdateCommand(uniqueKey, id, updateSpecs, parameters);
+            return SendAndParseHeader(atomicUpdate);
+        }

```

Comment:
Also in this interface, each Synchronous method has an Asynchronous version, so could you add the Async methods?

Reply:
This comment was meant for ISolrOperations